src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -41,7 +46,113 @@ public void InterpreterTestCase(ExpressionTestCase testCase)
                     Skip.If(true, prefix + msg);
                     break;
             }
-        } 
+        }        
+
+        [InterpreterTheory] //(Skip="NotReadyYet")]
```

Comment:
>InterpreterTheory [](http://example.com/codeflow?start=9&length=17)

Is the intention to check this in? Or just keep for private testing?

Reply:
I was thinking at private testing but we can check that in with Skip="NotReadyYet" while we fix all functions

Reply:
I'll remove that code before checking this in